frontstage/controllers/party_controller.py

```diff
@@ -324,8 +334,7 @@ def get_survey_list_details_for_party(party_id, tag, business_party_id, survey_i
     caching_data_for_survey_list(cache_data, surveys_ids, business_ids, tag)
     caching_data_for_collection_instrument(cache_data)
 
-    for enrolment in get_respondent_enrolments(party_id):
```

Comment:
Before this was calling the same function twice within a few lines of each other?  Ew.  Your change makes this much better